### PR TITLE
Update mdx_latex.py

### PR DESCRIPTION
**Pull request by sajozsattila**
add 'smarty' and 'sane_lists' support

### CODE REVIEW

File: mdx_latex.py [hunk 1]
```diff
@@ -172,7 +193,7 @@ def tolatex(self, ournode):
         subcontent = ""
 
         if ournode.text:
-            subcontent += escape_latex_entities(ournode.text)
+            subcontent += escape_latex_entities(ournode.text)
```
Comment (rufuspollock) [inline mdx_latex.py:196]: Can you fix the trailing whitespace ...

File: mdx_latex.py [hunk 2]
```diff
@@ -193,16 +214,21 @@ def tolatex(self, ournode):
             buffer += '\n\\subsubsection{%s}\n' % subcontent
         elif ournode.tag == 'hr':
             buffer += '\\noindent\makebox[\linewidth]{\\rule{\linewidth}{0.4pt}}'
-        elif ournode.tag == 'ul':
+        elif ournode.tag == 'ul':
```
Comment (rufuspollock) [inline mdx_latex.py:217]: Ditto

**Review by rufuspollock — CHANGES_REQUESTED** [review]
Thanks and lgtm in general. can you remove the trailing whitespace?

**Comment by sajozsattila** [comment]
Done

**Comment by rufuspollock** [comment]
Thank-you for the contribution.

**Pull request merged**